runtime/near-vm-logic/src/logic.rs

```diff
@@ -1137,6 +1137,13 @@ impl<'a> VMLogic<'a> {
     ///
     /// `input_cost(num_bytes_signature) + input_cost(num_bytes_message) + input_cost(num_bytes_public_key) +
     ///  ed25519_verify_base + ed25519_verify_byte * num_bytes_message`
+    ///
+    /// # Error
+    ///
+    /// If the public key's size is not equal to 32 returns [HostError::Ed25519VerifyInvalidInput].
+    /// Also if the signature size is not equal to 64 or the signature's "s" scalar isn't properly reduced
+    /// then returns [HostError::Ed25519VerifyInvalidInput].
```

Comment:
```suggestion
    /// If the signature size is not equal to 64  returns [HostError::Ed25519VerifyInvalidInput].
```

I think that's what NEP and impl do, right?